### ISSUE TITLE
per-atom-charge and per-atom-spin loss

### ISSUE THREAD

**Issue by chrisiacovella** [choderalab/modelforge#359]
We should implement a per-atom-charge and per-atom-spin loss function. 

For the per-atom partial charges, we have indirectly used these in the loss function via total charge and dipole moment.  We had purposefully avoided partial charges directly, because we assumed it would likely just be very noisy.  However, it should be relatively straightforward to implement and per-atom spin would use the same basic scheme. 
 
Correctly predicting per-atom spin might be very important to the transition metals, as it should help to identify the region around the metal center impacted by its presence which might prove crucial for training.

**Comment by chrisiacovella** [comment]
per atom charge has been added in PR #361
